src/serde.rs

```diff
@@ -60,6 +60,17 @@ impl<'de> Deserialize<'de> for Level {
 
                 self.visit_str(variant)
             }
+
+            fn visit_u32<E>(self, v: u32) -> Result<Self::Value, E>
```

Comment:
It looks like `serde` uses `visit_u64` for this, which makes sense, since `visit_u32` forwards to it by default, so let's target `visit_u64` instead.

Reply:
Ah, I see your second commit changes this to `visit_u32`. I think sticking with the more compatible `visit_u64` like you originally had is a good idea :+1:

Reply:
Done